orchestrator-salt/src/main/resources/salt-common/pillar/fluent/init.sls

```diff
@@ -6,16 +6,23 @@ fluent:
   agentLogFolderPrefix: /var/log
   serviceLogFolderPrefix: /var/log
   platform:
+  region:
```

Comment:
I think this will remain uninitialized due to `FluentConfigView.Builder.withRegion()` not being called anywhere.

Reply:
that's true, what we are doing in fluentConfigView, all the properties can be overriden by fluentAttributes, the withRegion should be used if we would try to support cloudwatch from non-aws envs..i can remove the with method but that field will be a bit hidden

Reply:
i think it's just better to have that as an option in cloudwatch params (by default if it's not there, that is calculated but only on aws platform)